devutils/develop-libraries.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+
+# Save current directory and change to the script's directory
+pushd . > /dev/null
+scriptPath=$(dirname "$0")
+cd "$scriptPath"
+
+# Get RCC
+if [ ! -f "./rcc" ]; then
+    curl -o rcc https://downloads.robocorp.com/rcc/releases/latest/linux64/rcc
+    chmod +x rcc
+fi
+
+# Go to repo root
+cd ..
+
+# Check if activate script exists
+if [ -f "./venv" ]; then
```

Comment:
What should happen if there's no venv at all in place?

Reply:
The windows script builds the venv if there is none, without questions. The mac/linux script is optional, if no one needs it we do not need to add it right now.

Reply:
Fixed the script to build a venv in case there's no existing one at all.